packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const questionnaireSubmittedController = require('../controllers/questionnaire-submitted');
+const fetchExistingAppealReplyMiddleware = require('../middleware/fetch-existing-appeal-reply');
+const fetchAppealMiddleware = require('../middleware/fetch-appeal');
+
+const router = express.Router();
+
+router.get(
+  '/:id/questionnaire-submitted',
+  // Currently unknown where email address will come from, so keeping both middleware
+  [fetchAppealMiddleware, fetchExistingAppealReplyMiddleware],
```

Comment:
Are these middlewares needed here? Could save a calll to the APIs/session

Reply:
Was leaving in awaiting email address task. Suppose not necessary yet.

Reply:
makes sense. In future will only need one of these anyway